src/main/java/org/jenkinsci/test/acceptance/controller/WinstoneController.java

```diff
@@ -47,8 +47,12 @@ public void addJavaOpt(String javaOpt) {
 
     @Inject
     public WinstoneController(Injector i) {
+        this(i, IOUtil.randomTcpPort());
+    }
+
+    public WinstoneController(Injector i, int httpPort) {
```

Comment:
do you need `@Inject` on this as well?

Reply:
I don't need it because this is being used by subclasses.